app/app.g.less

```diff
@@ -368,7 +368,8 @@ select {
 }
 
 .v--modal {
-  background-color: @day-bg;
+  background-color: var(--background);
+  box-shadow: 0 0 0 var(--shadow);
```

Comment:
Does this do anything? Is it animated later?

Reply:
not sure, just trying to change it the least from staging